src/dc_receive_imf.rs

```diff
@@ -1917,15 +2044,25 @@ pub(crate) async fn get_prefetch_parent_message(
     Ok(None)
 }
 
+/// * param `prevent_rename`: if true, the display_name of this contact will not be changed. Useful for
+/// mailing lists: In mailing lists, many users write from the same address but with different
```

Comment:
This is not true for all mailing lists. Some pass the `From:` header through, effectively spoofing it.

Maybe say "In some mailing lists" here.